bin/robots_check.sh

```diff
@@ -4,7 +4,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-if curl -s https://${HOSTNAME}/robots.txt | grep -q "Sitemap: https://www.mozilla.org/sitemap.xml";
+if curl -s https://${HOSTNAME}/robots.txt | grep -q "disallow: /*/whatsnew/";
```

Comment:
This change is fine if this check is only for the prod domain, otherwise it's not. It looks like it only uses the `MOZORG_CDN_HOSTNAME` in secrets, but it's in secrets for some reason so I can't check the value.

Reply:
Yeah MOZORG_CDN_HOSTNAME is the public-facing prod domain for sure. TBH we could have that one outside of secrets and keep the origins in secrets (even though the origins are not super top sekrit anyway). I made the decision to put them all in secrets many moons ago...